site/src/components/MetricsPreview.tsx

```diff
@@ -132,205 +147,250 @@ const MetricsPreview = () => {
   const lineHeightNormal = lineHeightScale * previewFontSize;
 
   return (
-    <Box
-      display="flex"
-      flexDirection="column"
-      justifyContent="center"
-      alignItems="center"
-      pos="relative"
-      paddingBottom={!editMetrics ? [16, 16, 0] : undefined}
-    >
-      {!editMetrics && (
-        <IconButton
-          icon={<EditIcon />}
-          aria-label="Customise font metrics"
-          title="Customise font metrics"
-          variant="outline"
-          borderRadius={20}
-          _hover={{ color: 'pink.500', background: 'transparent' }}
-          _focus={{ boxShadow: 'outline', borderColor: 'transparent' }}
-          _active={{ transform: 'scale(.9)' }}
-          onClick={() => setEditMetrics(true)}
-          color="gray.600"
-          pos="absolute"
-          bottom={0}
-          right={0}
-        />
-      )}
-      <Box maxWidth="100%" display="flex" justifyContent="center">
-        <Box
-          fontSize={previewFontSize}
-          fontFamily={
-            selectedFont.name.indexOf(' ') > -1
-              ? `'${selectedFont.name}'`
-              : selectedFont.name
-          }
-          lineHeight="normal"
-          pos="relative"
-          overflow="auto"
-          paddingLeft="130px" // cater for arrow offsets
-          paddingRight="150px" // cater for arrow offsets
-          paddingTop="40px" // cater for ascender overflow
-          marginTop="-40px" // cater for ascender overflow
-          paddingBottom="60px" // cater for descender overflow
-          marginBottom="-60px" // cater for descender overflow
-        >
-          <Box display="inline-flex" justifyContent="center" pos="relative">
-            <Box
-              pos="absolute"
-              top={0}
-              bottom={0}
-              right={0}
-              left={0}
-              bg="pink.200"
-              opacity={0.3}
-            />
-            <Metric
-              position={previewFontSize}
-              hoffset={20}
-              voffset={(lineHeightNormal - previewFontSize) / 2}
-              label={`Em square (${metrics.unitsPerEm})`}
-              align="right"
-              guides="all"
-            />
+    <Tabs size="sm" variant="soft-rounded" align="center" colorScheme="gray">
+      <TabList position="relative" zIndex={1} mb={8}>
+        <Tab>
+          <Box as="span" mr="2" mt="1">
+            <Icon as={TbRulerMeasure} />
+          </Box>
+          Measurements
+        </Tab>
+        <Tab>
+          <Box as="span" mr="2" mt="1">
+            <Icon as={VscJson} />
+          </Box>{' '}
```

Comment:
Was the space here intentional?
```suggestion
          </Box>
```